src/main/java/org/jabref/gui/preferences/customentrytypes/CustomEntryTypesTab.java

```diff
@@ -176,7 +177,25 @@ private void setupFieldsTable() {
         makeRotatedColumnHeader(fieldTypeColumn, Localization.lang("Required"));
 
         fieldTypeMultilineColumn.setCellFactory(CheckBoxTableCell.forTableColumn(fieldTypeMultilineColumn));
-        fieldTypeMultilineColumn.setCellValueFactory(item -> item.getValue().multilineProperty());
+        fieldTypeMultilineColumn.setCellValueFactory(item -> {
+            BooleanProperty property = item.getValue().multilineProperty();
+            property.addListener((obs, wasSelected, isSelected) -> {
+                if (isSelected) {
+                    viewModel.entryTypes().forEach(typeViewModel -> {
+                        typeViewModel.fields().stream()
+                                     .filter(f -> f.displayNameProperty().get().equals(item.getValue().displayNameProperty().get()))
+                                     .forEach(f -> f.multilineProperty().set(true));
+                    });
+                } else {
+                    viewModel.entryTypes().forEach(typeViewModel -> {
+                        typeViewModel.fields().stream()
+                                     .filter(f -> f.displayNameProperty().get().equals(item.getValue().displayNameProperty().get()))
+                                     .forEach(f -> f.multilineProperty().set(false));
+                    });
+                }
+            });
+            return property;
```

Comment:
For the sake of clean code, please extract this to a seperate method.
Also please do not use `f` as a var name but `field`.

Reply:
My latest changes on commit 1ca84f7b63fa2a2d1652a7dd35ed3fec82c172cd extracts this into a separate method and removes the `f` abbreviation replacing it with var name `field` . Thanks!